vms/platformvm/txs/builder/builder.go

```diff
@@ -294,93 +276,24 @@ func (b *builder) NewImportTx(
 	changeAddr ids.ShortID,
 	memo []byte,
 ) (*txs.Tx, error) {
-	kc := secp256k1fx.NewKeychain(keys...)
```

Comment:
This is the main change of the PR. The logic in builder is simply dropped and the wallet builder and signer are used instead.